frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -2753,7 +2753,7 @@ resolveFunctionByInfoQuery(ResolutionContext* rc,
   CHPL_RESOLUTION_QUERY_BEGIN(resolveFunctionByInfoQuery, rc, sig, poiInfo);
 
   // Call the implementation which resolves the function body.
-  auto resolved = resolveFunctionByInfoImpl(rc, sig, std::move(poiInfo));
+  auto resolved = resolveFunctionByInfoImpl(rc, sig, poiInfo);
```

Comment:
why change this?

Reply:
Because `poiInfo` is used below, so the move is not sound :)